fetch.bzl

```diff
@@ -31,6 +31,17 @@ def fetch_images():
         ],
     )
 
+    # ECR public registry
```

Comment:
does this one require the credential helper? add a comment so we know it's testing that case

Reply:
yes, it works as we know how to authenticate against public ECR. but when --credential_helper goes into affect, the authentication is done by the auth helper instead of oci_pull. I'll this as comment.